Author: ljlane
Description: libnetfilter-acct source does not include
  a changelog. This is a compilation of the external changelog
  files from http://netfilter.org/projects/libnetfilter_acct/files/

--- /dev/null
+++ b/Changelog
@@ -0,0 +1,51 @@
+changes-libnetfilter_acct-1.0.1.txt:
+
+Jan Engelhardt (1):
+      build: remove unnecessary pkgconfig->config.status dependency
+
+Pablo Neira Ayuso (4):
+      src: NFACCT_SNPRINTF_T_XML flag for nfacct_snprintf to output time
+      build: bump version to 1.0.1
+      src: remove unnecessary castings
+      src: NFACCT_PKTS and NFACCT_BYTES are MNL_TYPE_U64
+
+
+
+changes-libnetfilter_acct-1.0.0.txt:
+
+Jan Engelhardt (5):
+      build: enable automake warnings
+      build: remove unnecessary AC_EXEEXT
+      Update .gitignore
+      Add examples/.gitignore
+      build: add missing LIBMNL_CFLAGS to compilation process
+
+Pablo Neira Ayuso (27):
+      intial import of libnetfilter_acct
+      fix crash if no data pointer is passed to nfacct_list_cb
+      src: improve nfacct-add example
+      add missing .gitignore
+      add missing Makefile.am files
+      update headers to use new nfnetlink subsys
+      examples: nfacct-get: display pkt and byte counters
+      fix endianness in 64-bits packet and bytes counters
+      examples: nfacct-add: initialize counters, otherwise we add garbage
+      add README file
+      add NFNL_MSG_ACCT_GET_CTRZERO
+      examples: add nfacct-del to delete accounting objects
+      extend nfacct_list to allow to zero counters
+      examples: nfacct-get allows to zero counters with `-z' option
+      update nfnetlink_acct.h
+      src: major API redesign
+      src: add library documentation in doxygen
+      add COPYING file and other licensing stuff
+      update README file
+      build: update configure.ac and gcc4_visibility.m4
+      update .gitignore file
+      nfacct: modify nfacct_snprint output
+      nfacct: use NFACCT_ATTR_* flags in nfacct_nlmsg_build_payload
+      nfacct: check if attribute is set in nfacct_attr_get*
+      extend README
+      nfacct: change prototype of nfacct_snprintf() to support output in XML
+      examples: fix nfacct-get example
+
